newsletter-march-2023.rst

```diff
@@ -0,0 +1,127 @@
+.. post:: March 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - March 2023
+=====================================
+
+News and updates
+----------------
+
+Here are the latest updates from our team since the :doc:`previous newsletter </newsletter-february-2023>`:
+
+- ⭐️ We passed our 10,000th issue/pull request on GitHub. And it's pretty much an equal split between the `5039 issues <https://github.com/readthedocs/readthedocs.org/issues>`__ and `4872 pull requests <https://github.com/readthedocs/readthedocs.org/pulls>`__ now registered.
+  Thanks to the whole community for building this together through code, issues, suggestions... and documentation!
+- 🌪️ Follow up: Build times have gone rapidly down after last month's introduction of parallel uploading of artifacts with `rclone`_.
+  Depending on the number of files in a build outputs,
+  build times may have gone down several seconds or several minutes.
+  For instance, a large project like `Write the Docs <https://readthedocs.org/projects/writethedocs-www/builds/>`__ has gone down from ~7 minutes to under 3 minutes.
+  If you want to see the change for your project,
+  have a look at your build times before and after **February 8**.
+- 1️⃣️ We are going to have 1 main Read the Docs website in the future.
+  The first move has been done now,
+  as readthedocs.com now points to `about.readthedocs.com <https://about.readthedocs.com>`__ instead of being a separate site for Read the Docs for Business.
+  The separate landing page on readthedocs.org will also be moving.
```

Comment:
```suggestion
- 1️⃣️ We are going to have 1 primary marketing website in the future,
while still running two different product for our Community & Business users.
  The first change that is now done is redirecting logged out users on `https://readthedocs.com/` to `about.readthedocs.com <https://about.readthedocs.com>`__.
  We will soon be doing this for logged out users going to `https://readthedocs.org/`,
  but not it's project-specific subpages.
```

Seems like an area where your suggestion to be very explicit is useful. I think we should probably have a link here explaining this process.

Reply:
We have a blog post now that we can link 😉

Reply:
Yes, haha.. luxury problem: we'll need to resolve the order of how these are published. Ideally, we get the other blog post out first so it can be referenced here... but I think that the other blog post will have its own spotlight (mailing list + twitter post) and maybe even have more clicks than the newsletter, so it's not too important :)

Reply:
Yea, we shouldn't block this post on that post